src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -19,6 +19,8 @@
   --pf-c-dropdown__toggle--hover--BorderBottomColor: var(--pf-global--active-color--100);
```

Comment:
can you add `--pf-c-dropdown__toggle--Color: var(--pf-global--Color--100);` (then set the `color` property) so that when you toggle the dropdown in a dark theme, the toggle text updates?

And on L255, we're redefining `--pf-c-dropdown__menu-item--Color` to `var(--pf-c-dropdown__menu-item--Color)` which doesn't do anything :) Can you change that to `--pf-c-dropdown__menu-item--Color: var(--pf-c-dropdown__menu-item--hover--Color);` and then add `  --pf-c-dropdown__menu-item--hover--Color: var(--pf-global--Color--dark-100);`. That will also fix https://github.com/patternfly/patternfly-next/issues/1286

Reply:
Great! It works now.